R/FilterState.R

```diff
@@ -464,6 +463,7 @@ InteractiveFilterState <- R6::R6Class( # nolint
     dataname = character(0),
     varname = character(0),
     choices = NULL, # because each class has different choices type
+    choices_limited = FALSE, # flag whether number of possible choices was limited when specifying filter
```

Comment:
`choices_limited` suggest that this field contains choices

```suggestion
    is_choices_limited = FALSE, # flag whether number of possible choices was limited when specifying filter
```